src/systems/filecoin_markets/retrieval_market/retrieval_provider.md

```diff
@@ -2,4 +2,15 @@
 title: "Retrieval Provider (Miner)"
 ---
 
+### Provider Dependencies
+
+The Retrieval Provider Depends On The Following Dependencies
+**Host**: A libp2p host (set setup the libp2p protocols)
+**Node**: A node implementation to query the chain for pieces and to setup and manage payment channels
+**StorageMiner**: For unsealing sectors
```

Comment:
This is fine for now but I think we will define what a StorageMinerNode is (which contains StorageMiningSubsystem that will include the functionality to unseal sectors. Similarly, a Node will become a Filecoin Node.